redis-utils/src/main/java/org/ballerinalang/redis/utils/RedisDbUtils.java

```diff
@@ -46,22 +52,69 @@
      *
      * @throws IOException exception
      */
-    public static void initServer() throws IOException {
-        String macExecutablePath = Paths.get(System.getProperty("user.dir")).resolve("src").resolve("redis").
-                resolve("tests").resolve("resources").resolve("redis-executable").resolve("redis-server-5.0.7-mac")
-                .toString();
-        String linuxExecutablePath = Paths.get(System.getProperty("user.dir")).resolve("src").resolve("redis").
-                resolve("tests").resolve("resources").resolve("redis-executable").resolve("redis-server-5.0.7-linux")
-                .toString();
-        redisServer = new CustomRedisServer(macExecutablePath, linuxExecutablePath, REDIS_PORT);
-        redisServer.start();
+    public static Object initServer() throws IOException {
+        BufferedReader reader = null;
```

Comment:
Shall we use this try with resources statement?

Reply:
Update in fb3d98e